src/components/dialogs/create-case-dialog.js

```diff
@@ -40,14 +40,12 @@ export function CreateCaseDialog({ onClose, open }) {
     const activeDirectory = useSelector((state) => state.activeDirectory);
     const userId = useSelector((state) => state.user.profile.sub);
     const dispatch = useDispatch();
-    const [triggerReset, setTriggerReset] = useState(true);
     const [name, NameField, nameError, nameOk, setCaseName, touched] =
         useNameField({
             label: 'nameProperty',
             autoFocus: true,
             elementType: ElementType.CASE,
             parentDirectoryId: activeDirectory,
-            triggerReset,
```

Comment:
but then this field is no longer reset ?

Reply:
It was never reset except when the dialog was closed, and now it's unmounted